### PR TITLE
remove_deprecated_for_portal-kernel_part_Eleven

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

This is remove_deprecated_for_portal_kernel_part_Eleven.
can also see https://github.com/jeffwu0724/liferay-portal/pull/58.
Thanks for checking.

### CODE REVIEW

**Comment by jeffwu0724** [comment]
@catarineaguiar Can you help me look at the test result, please?

Thanks
Jeff

**Comment by catarineaguiar** [comment]
Hi @jeffwu0724 there is a semver error...

```
[exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/portal-kernel/portal-kernel.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/48.0.0/50b749de920d78a987725a212a2a3e6886d684fa/com.liferay.portal.kernel-48.0.0.jar
```
full log [here](https://testray.liferay.com/reports/production/logs/2022-04/test-1-37/test-portal-acceptance-pullrequest(master)/1953/semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

File: portal-impl/test/unit/com/liferay/portal/kernel/flash/FlashMagicBytesUtilTest.java [hunk 1]
```diff
@@ -16,16 +16,8 @@
 
 import com.liferay.portal.test.rule.LiferayUnitTestRule;
 
-import java.io.ByteArrayInputStream;
```
Comment (tinatian) [inline portal-impl/test/unit/com/liferay/portal/kernel/flash/FlashMagicBytesUtilTest.java:19]: @jeffwu0724 The FlashMagicBytesUtil is removed, we do not need the FlashMagicBytesUtilTest at all, please remove it.

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test

**Comment by jeffwu0724** [comment]
@catarineaguiar Can you help me look at the test result again, please?

Thanks
Jeff

**Comment by catarineaguiar** [comment]
@jeffwu0724 no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/117164

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**